apps/mochi-web/components/app/detail/AppDetailStatistics.tsx

```diff
@@ -122,6 +122,11 @@ export const AppDetailStatistics = ({
                       className="text-sm font-medium"
                       disabled={editing !== field.name}
                       onBlur={() => setEditing('')}
+                      onPaste={(e) => {
+                        e.preventDefault()
+                        const text = e.clipboardData.getData('text/plain')
+                        document.execCommand('insertText', false, text)
+                      }}
```

Comment:
What caused the problem? This fix is only here or we need to add the similar code in another place in the future?

Reply:
the paste event includes html tag so it makes the text longer, maybe we will need similar code to exclude the html tag

Reply:
Should we add a prop to switch on/off the behavior? And I think the default should be off. In that case, consider using https://react.toolkit.d.foundation/utils/call-all-handlers to merge the paste events.